pageserver/src/page_service.rs

```diff
@@ -315,6 +326,14 @@ impl PageServerHandler {
     }
 }
 
+lazy_static! {
+    static ref PROCESS_QUERY_COUNT: IntCounter = register_int_counter!(
```

Comment:
That one is not extremely useful. Our most interesting query is getPage, but it happens inside of 'pagestream' command (that switches libpq protocol to copyboth state). So even if we serve 1M pages in a given connection that would lead to a single increment of that counter.

Let's implement SMGR_QUERY_COUNTER parametrized by smgr operation instead.